### PR TITLE
Code improvements about semantics and best practices

### PR DESCRIPTION
**Pull request by rhuan080**
Signed-off-by: Rhuan Rocha <rhuan080@gmail.com>

### CODE REVIEW

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

File: tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/data/PetData.java [hunk 1]
```diff
@@ -13,14 +13,12 @@
...
+import org.apache.commons.lang3.StringUtils;
 import org.eclipse.microprofile.openapi.apps.petstore.model.Category;
 import org.eclipse.microprofile.openapi.apps.petstore.model.Pet;
 import org.eclipse.microprofile.openapi.apps.petstore.model.Tag;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import java.util.HashMap;
+import java.util.*;
```
Comment (arthurdm) [inline tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/data/PetData.java]: do we really need to import everything from that package?

Comment (EricWittmann) [reply tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/data/PetData.java]: Definitely not. :)

Comment (rhuan080) [reply tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/data/PetData.java]: Hi, I'm so sorry. It is not needed.

Comment (rhuan080) [reply tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/data/PetData.java]: I'll fix it.

Comment (rhuan080) [reply tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/data/PetData.java]: Updated.

**Comment by MikeEdgar** [comment]
I would personally avoid using the `StringUtils` `isBlank` and `isNotBlank` methods for a couple of reasons. First, because it increases the direct dependency on commons-lang3 for a trivial piece of functionality. Second, because it is logically different than the code it's replacing in this case.

* `status != null && !"".equals(status)` is only true if the string is non-zero length (i.e. whitespace would be allowed)
* `StringUtils.isNotBlank(status)` is true only for non-whitespace characters.

I would suggest using `isEmpty` on the `String` instance after the `null` check instead of the `StringUtils` method calls.

**Comment by rhuan080** [comment]
--> Second, because it is logically different than the code it's replacing in this case.
In this code snippet the status should not be a whitespace. 

--> First, because it increases the direct dependency on commons-lang3 for a trivial piece of functionality.
I think it is not a problem because commons-lang3 is a well known lib that is used a lot.

**Pull request closed without merging**